koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinApplication.kt

```diff
@@ -35,19 +35,27 @@ class KoinApplication private constructor() {
     internal fun loadDefaults() {
         koin.scopeRegistry.loadDefaultScopes(koin)
     }
-
+    
     /**
-     * Load definitions from modules
+     * Load definitions from a single module
      * @param modules
      */
     fun modules(modules: Module): KoinApplication {
```

Comment:
You should probably update the name of the parameter here:
```suggestion
    fun modules(module: Module): KoinApplication {
```

Reply:
yes

Reply:
please add this

Reply:
This was already completed in one of the previous [commits](https://github.com/InsertKoinIO/koin/pull/481/commits/ca701ebffbc572b5e62383588087faf5c1077bad)

Reply:
> This was already completed in one of the previous commits

I'm not sure I got your answer @TSurkis. My comment was about renaming the parameter to `module` (singular) as it's only one in this method overload.